parastell/magnet_coils.py

```diff
@@ -90,292 +144,223 @@ def logger(self):
-            if (min_phi >= min_rad or min_phi <= max_rad) or (
-                max_phi >= min_rad or max_phi <= max_rad
+            if (min_tor_ang >= lower_bound or min_tor_ang <= upper_bound) or (
+                max_tor_ang >= lower_bound or max_tor_ang <= upper_bound
             ):
-                reduced_fils.append(fil)
+                filtered_coords.append(coords)
+                filtered_tangents.append(tangents)
                 com_list.append(com)
 
-        reduced_fils = np.array(reduced_fils)
+        filtered_coords = np.array(filtered_coords)
+        filtered_tangents = np.array(filtered_tangents)
         com_list = np.array(com_list)
 
         # Compute toroidal angles of filament centers of mass
-        phi_arr = np.arctan2(com_list[:, 1], com_list[:, 0])
-        phi_arr = (phi_arr + 2 * np.pi) % (2 * np.pi)
+        com_toroidal_angles = np.arctan2(com_list[:, 1], com_list[:, 0])
+        com_toroidal_angles = (com_toroidal_angles + 2 * np.pi) % (2 * np.pi)
 
-        # Sort filaments by toroidal angle
-        self.filtered_filaments = [
-            x for _, x in sorted(zip(phi_arr, reduced_fils))
-        ]
+        # Sort filaments by toroidal angle and overwrite respective arrays
+        self.filament_coords = np.array(
+            [x for _, x in sorted(zip(com_toroidal_angles, filtered_coords))]
+        )
+        self.filament_tangents = np.array(
+            [x for _, x in sorted(zip(com_toroidal_angles, filtered_tangents))]
+        )
+        self.filament_com = np.array(
+            [x for _, x in sorted(zip(com_toroidal_angles, com_list))]
+        )
 
-    def _cut_magnets(self, volume_ids):
-        """Cleanly cuts the magnets at the planes defining the toriodal extent.
+    def _cut_magnets(self):
+        """Cuts the magnets at the planes defining the toriodal extent.
         (Internal function not intended to be called externally)
-
-        Arguments:
-            volume_ids (list): volume ids corresponding to each magnet volume
-
-        Returns:
-            volume_ids (range): new volume ids corresponding to magnet volumes
-                following cutting operation
         """
-        # Define sweeping surface width
-        # Multiply by factor of 2 to be conservative
-        rec_width = 2 * self.average_radial_distance
-
-        cubit.cmd(f"create surface rectangle width {rec_width} yplane")
-        surf_id = cubit.get_last_id("surface")
-
-        # Shift surface to positive x axis
-        cubit.cmd(f"move Surface {surf_id} x {rec_width/2}")
-
-        # Revolve surface to create wedge spanning toroidal extent
-        cubit.cmd(
-            (
-                f"sweep surface {surf_id} zaxis angle "
-                f"{np.rad2deg(self._toroidal_extent)}"
-            )
+        toroidal_region = cq.Workplane("XZ")
+        toroidal_region = toroidal_region.transformed(
+            offset=(1.25 * self.max_radial_distance / 2, 0)
```

Comment:
It seems useful to assign `1.25 * self.max_radial_distance` to a variable and reuse it throughout